recipes/agg-regrid/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "agg-regrid" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "1ab034cd9c5363afcd2cf74073bf11ab16d9e3c81729433eb008fcb6e81be976" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/SciTools-incubator/iris-{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [osx]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
```

Comment:
Tests passed because `iris` includes `numpy` at runtime, but this package links to `numpy`, and so should need to [specify version numbers](https://conda-forge.org/docs/meta.html#building-against-numpy), and add it as a runtime dependency....